ITEvents/Screens/Favorive tab/FavoritePresenter.swift

```diff
@@ -4,31 +4,41 @@ class FavoritePresenter: IFavoritePresenter {
     var view: IFavoriveView!
     var eventDataService: IEventsDataService!
     var selectedEventService: ISelectedEventService!
-    private var isListLayoutSelected = true
+    var userSettingsService: IUserSettingsService!
+    private var userSettings: UserSettings!
     
     init(view: IFavoriveView,
          eventDataService: IEventsDataService,
-         selectedEventService: ISelectedEventService) {
+         selectedEventService: ISelectedEventService,
+         userSettingsService: IUserSettingsService) {
         self.view = view
         self.eventDataService = eventDataService
         self.selectedEventService = selectedEventService
+        self.userSettingsService = userSettingsService
     }
     
     func setup(then completion: @escaping () -> Void) {
+        userSettingsService.fetchSettings { settings in
+            self.userSettings = settings
```

Comment:
The following line should be moved from the bottom here (into fetchSettings completion callback)
```
view.toggleListLayout(to: settings.isListLayoutSelected)
```